src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -521,6 +522,14 @@ void AddBehavior(string tag, BehaviorModule behavior)
                 AddBehavior("ModuleTag_ExperienceHelper", new ExperienceUpdate(this));
             }
 
+            // TODO: This shouldn't be added to all objects. I don't know what the rule is.
+            if (_gameContext.Game.SageGame >= SageGame.CncGeneralsZeroHour)
+            {
+                AddBehavior("ModuleTag_StatusDamageHelper", new StatusDamageHelper());
+                AddBehavior("ModuleTag_SubdualDamageHelper", new SubdualDamageHelper());
```

Comment:
I don't know why, but I had these two added before `ModuleTag_DefectionHelper`. Maybe that was the order they were in? Or maybe I was drunk when I wrote that code 🤷

Reply:
https://github.com/OpenSAGE/OpenSAGE/commit/fcca96a81535183ca3a38cce4f67068449c8b89e